samples/boards/96b_argonkey/microphone/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
Move this below the license/copyright header

Reply:
ack (did it also for CMakelist.txt in sensor)